### PR TITLE
Switch from centos7-os to centos-7-server external repositories

### PR DESCRIPTION
**Pull request by ehelms**
Koji has duplicate external repositories for CentOS 7 Server and
Updates and this consolidates the use.

### CODE REVIEW

**Comment by ehelms** [comment]
Needs this change https://github.com/theforeman/foreman-infra/pull/1614/files#diff-5785de757e436cf61a963a490cdad9051c83b296873434569fc8a2733406042fR16 to be able to merge so that the tags when updated will behave the same.

File: configs/katello/3.15.yaml [hunk 1]
```diff
@@ -80,5 +80,5 @@
         katello-pulpcore-3.15-el7: 0
     build_package_group_source_tag: katello-pulpcore-nightly-el7-build
     external_repos:
-      - centos7-os
-      - centos7-updates
+      - centos-7-server
```
Comment (ekohl) [inline configs/katello/3.15.yaml:83]: Can we standardize on `centos-7-os` instead? That's the repo name that CentOS actually uses (as well as updates). There is no such thing as CentOS Server.

Comment (ehelms) [reply configs/katello/3.15.yaml:83]: I can do that as a follow up step, once everything is using a single repository, renames become easier.

Comment (ekohl) [reply configs/katello/3.15.yaml:83]: If anything, I think we should remove `centos-7-server` then. Note you are renaming updates as well here by adding a dash.

Comment (ehelms) [reply configs/katello/3.15.yaml:83]: Right now, I want to aim for alignment first (and removing duplication). Then do mass updates. The change to remove `centos-7-server` would at present touch 187 tags:

```
$ grep -r centos-7-server configs/ | wc -l
187
```

Now we wont actually need to update all of them but I want to give an indicator of my preferred strategy here.

File: configs/katello/3.15.yaml [hunk 2]
```diff
@@ -80,5 +80,5 @@
...
+      - centos-7-updates
```
Comment (ekohl) [inline configs/katello/3.15.yaml:84]: Today this doesn't exist. Do you intend to rename it?

Comment (ehelms) [reply configs/katello/3.15.yaml]: Vim mass update fail, this is actually named `centos-7-server-updates` and should now be reflected.

**Comment by ekohl** [comment]
@ehelms given we're moving to Copr I'm going to close this.

**Pull request closed without merging**